install/operator/deploy/crds/syndesis.io_syndesis_crd.yaml

```diff
@@ -2226,6 +2226,36 @@ spec:
                     type: object
                   server:
                     properties:
+                      connectionPool:
+                        description: Connectin Pool parameters used in syndesis-server
```

Comment:
Connection pool - and we probably want to remove the link reference to hikariCP

Reply:
Fixed. Thanks.